systers_portal/meetup/views.py

```diff
@@ -19,3 +20,18 @@ class MeetupLocationList(ListView):
     template_name = "meetup/list_location.html"
     model = MeetupLocation
     paginate_by = 20
+
+
+class MeetupView(MeetupLocationMixin, DetailView):
```

Comment:
It might be requiring `model` attribute to be set up. Try to set it and see if it gives you the same error.